frontend/js/instructions.js

```diff
@@ -0,0 +1,54 @@
+const renderInstructionsHtml = (params) => {
+  setActiveSelectedMenu('instructions');
+  return (
+    `
```

Comment:
In general, it's a very very bad idea to have HTML components embed inside a js file because it is using a completely different engine to render this at runtime. You are much more susceptible to bugs. There are some ways to avoid this:
1. Separate .html file that you can read and embed into the main dom using javascript
2. HTML component inside the original html file with display: none. Use javascript to toggle css visibility. (I recommend doing this)